cmd/av/stack.go

```diff
@@ -91,10 +91,29 @@ var stackTreeCmd = &cobra.Command{
 	Use:   "tree",
 	Short: "show the tree of stacked branches",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return errors.New("unimplemented")
+		repo, err := getRepo()
+		if err != nil {
+			return err
+		}
+		trees, err := stacks.GetTrees(repo)
```

Comment:
I'm assuming there's no easy to cache this?

Reply:
I don't think it should be too computationally intensive. Think you'd need 1000s of stack branches for it to matter.

Though will need to make sure we're handling "garbage collection" smartly (removing stack metadata for branches that have been merged and whatnot).